src/pages/Team/team.module.scss

```
@@ -29,9 +53,103 @@
   }
 }
 
-.avatar {
-  width: 80px;
-  height: 80px;
-  background-color: #303030;
-  border-radius: 40px;
+.right {
+  flex-direction: row-reverse;
+  align-self: flex-end;
+  color: green;
+  h4 {
```

Добавь отступ между блоками

```suggestion
  
  h4 {
```